<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Test ct_telnet_SUITE:faulty_regexp result</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<a name="top"></a><pre>
=== Test case: <a href="ct_telnet_suite.src.html#faulty_regexp-1">ct_telnet_SUITE:faulty_regexp/1</a> (click for source code)

=== Config value:

    [{watchdog,&lt;0.112558.0&gt;},
     {trace_level,50},
     {ct_opts,[]},
     {ct_node,ct@f0b084c70b57},
     {tc_logfile,"/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/ct_telnet_suite.faulty_regexp.html"},
     {tc_group_properties,[{suite,ct_telnet_SUITE},{name,legacy}]},
     {tc_group_path,[]},
     {data_dir,"/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_telnet_SUITE_data/"},
     {priv_dir,"/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/"},
     {nodenames,[]}]

=== Current directory is "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32"

=== Started at 2024-10-09 20:08:16

<br />

</pre>
<div class="default"><pre><b>*** User 2024-10-09 20:08:16.210 ***</b><a id=e-992 class="link-to-entry" href="#e-992">&#x1f517;</a>
Testcase faulty_regexp starting!
</pre></div>
<pre>
See Common Test logs here:

<a href="file:///buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/all_runs.html">/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/all_runs.html</a>
<a href="file:///buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/index.html">/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/index.html</a>

Event receiver &lt;20819.803.0&gt; started!


[RUN #1] Calling ct:run_test([{logdir,
                                  "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/"},
                              {net_dir,[]},
                              {event_handler,
                                  {ct_test_support_eh,
                                      [{cbm,ct_test_support},
                                       {trace_level,50}]}},
                              {suite,
                                  "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_telnet_SUITE_data/ct_telnet_faulty_regexp_SUITE"},
                              {label,faulty_regexp},
                              {config,
                                  "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/telnet.faulty_regexp.legacy.cfg"}]) on ct@f0b084c70b57


Common Test starting (cwd is /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32)


Event Handler ct_test_support_eh started with [{cbm,ct_test_support},
                                               {trace_level,50}]


Common Test: Running make in test directories...

Recompile: ct_telnet_faulty_regexp_SUITE


CWD set to: "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/ct_run.ct@f0b084c70b57.2024-10-09_20.08.16"


TEST INFO: 1 test(s), 6 case(s) in 1 suite(s)


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: Starting test, 6 test cases


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 1 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 2 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 3 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 4 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 5 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 6 of 6 ***

Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: TEST COMPLETE, 0 ok, 6 failed of 6 test cases


Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/index.html ... 
done

Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/all_runs.html ... 
done


[RUN #1] Got return value {0,6,{0,0}} after 2020 ms

Saving start opts on ct@f0b084c70b57: [{halt_with,
                                        {ct_test_support,ct_test_halt}},
                                       {logdir,
                                        "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/"},
                                       {net_dir,[]},
                                       {event_handler,
                                        {ct_test_support_eh,
                                         [{cbm,ct_test_support},
                                          {trace_level,50}]}},
                                       {suite,
                                        "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_telnet_SUITE_data/ct_telnet_faulty_regexp_SUITE"},
                                       {label,faulty_regexp},
                                       {config,
                                        "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/telnet.faulty_regexp.legacy.cfg"}]

[RUN #2] Calling ct_run:script_start() on ct@f0b084c70b57


Common Test v1.27.2 starting (cwd is /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32)


Event Handler ct_test_support_eh started with [{cbm,ct_test_support},
                                               {trace_level,50}]


Common Test: Running make in test directories...


CWD set to: "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/ct_run.ct@f0b084c70b57.2024-10-09_20.08.18"


TEST INFO: 1 test(s), 6 case(s) in 1 suite(s)


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: Starting test, 6 test cases


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 1 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 2 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 3 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 4 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 5 of 6 ***


- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Error detected: "cth_conn_log:pre_init_per_testcase/4 CTH call failed"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: *** FAILED test case 6 of 6 ***

Testing common_test_test.ct_telnet_SUITE_data.ct_telnet_faulty_regexp_SUITE: TEST COMPLETE, 0 ok, 6 failed of 6 test cases


Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/index.html ... 
done

Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/all_runs.html ... 
done



[RUN #2] Got exit status value 1 after 3005 ms

Stopping event receiver!

Events written to logfile: <a href="file:///buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/faulty_regexp.events">/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@f0b084c70b57.2024-10-09_19.22.32/make_test_dir.common_test_test.logs/run.2024-10-09_19.22.35/log_private/faulty_regexp.events</a>

Found {ct_test_support_eh,start_logging,{'DEF','RUNDIR'}}!
(I) Failed to find {ct_test_support_eh,tc_done,
                    {ct_telnet_faulty_regexp_SUITE,expect_pattern,
                     {failed,
                      {error,
                       {{bad_pattern,"invalid(pattern",{"missing )",15}},
                        {ct_telnet,expect,3}}}}}} in the list of events!


</pre>
<div class="ct_error_notify"><pre><b>*** CT Error Notification 2024-10-09 20:08:21.244 ***</b><a id=e-994 class="link-to-entry" href="#e-994">&#x1f517;</a>
ct_telnet_SUITE:all_tests_in_suite failed on line 158
Reason: {badmatch,{event_not_found,
    {ct_test_support_eh,tc_done,...}

<a href="#end">Full error description and stacktrace</a>
</pre></div>
<pre>

</pre>
<div class="default"><pre><b>*** User 2024-10-09 20:08:21.245 ***</b><a id=e-995 class="link-to-entry" href="#e-995">&#x1f517;</a>
Stopping the telnet_server now!
</pre></div>
<pre>
[telnet_server, 2024-10-09 20:08:21.245]
telnet_server stopped

[telnet_server, 2024-10-09 20:08:21.245]
telnet_server closed the listen socket #Port&lt;0.309&gt;

<a name="end"></a>


=== Ended at 2024-10-09 20:08:22
=== Location: [{ct_telnet_SUITE,all_tests_in_suite,<a href="ct_telnet_suite.src.html#158">158</a>},
              {test_server,ts_tc,1794},
              {test_server,run_test_case_eval1,1303},
              {test_server,run_test_case_eval,1235}]
=== === Reason: no match of right hand side value 
                 {event_not_found,
                  {ct_test_support_eh,tc_done,
                   {ct_telnet_faulty_regexp_SUITE,expect_pattern,
                    {failed,
                     {error,
                      {{bad_pattern,"invalid(pattern",{"missing )",15}},
                       {ct_telnet,expect,3}}}}}}}
  in function  ct_telnet_SUITE:all_tests_in_suite/4 (ct_telnet_SUITE.erl, line 158)
  in call from test_server:ts_tc/3 (test_server.erl, line 1794)
  in call from test_server:run_test_case_eval1/6 (test_server.erl, line 1303)
  in call from test_server:run_test_case_eval/9 (test_server.erl, line 1235)
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Wed Oct 09 2024 20:08:16<!--/date--><br />
</div>
</center>
</body>
</html>
